gateway-service/src/main/java/org/zowe/apiml/gateway/controllers/AuthController.java

```diff
@@ -127,7 +114,7 @@ public ResponseEntity<String> revokeAccessToken(@RequestBody() Map<String, Strin
             return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
         }
         tokenProvider.invalidateToken(body.get(TOKEN_KEY));
-        return new ResponseEntity<>(HttpStatus.OK);
+        return new ResponseEntity<>(HttpStatus.NO_CONTENT);
```

Comment:
Why this change? it could be breaking change if we change the return code

Reply:
it is rather a bug fix, some clients could expect a body in case of 200 which could lead to stuck connections

Reply:
sure, I will do the change in zowe docs as well

Reply:
Thank you